.bazelversion

```diff
@@ -1 +1 @@
-3.3.1
+4.1.0
```

Comment:
Do we need this change? We have https://github.com/bazelbuild/rules_python/pull/533 bumping the version.

Reply:
that just landed, so no we shouldn't need this anymore